screens/LiveStreamScreen.tsx

```diff
@@ -0,0 +1,177 @@
+        }
+    },
+    body: {
+        ...Style.body, ...{
+        },
+    },
+})
+
+interface Props {
+    navigation: StackNavigationProp<HomeStackParamList | MainStackParamList, "NotesScreen">;
+    route: RouteProp<HomeStackParamList, 'NotesScreen'>;
+}
+
+type LiveEvent = {
+    id: string | null,
+    date: string | null,
+    startTime: string | null,
+    videoStartTime: string | null,
+    endTime: string | null,
+    prerollYoutubeId: string | null,
+    liveYoutubeId: string | null,
+} | null;
+
+export default function LiveStreamScreen(props: Props): JSX.Element {
+    const [currentEvent, setcurrentEvent] = useState<LiveEvent>(null);
+    const [showTime, setshowTime] = useState<boolean>(false)
+    //const mediaContext = useContext(MediaContext);
+    const playerRef = useRef<YoutubeIframeRef>(null);
+    const deviceWidth = Dimensions.get('window').width
+    const today = moment().utcOffset(moment().isDST() ? '-0400' : '-0500').format('YYYY-MM-DD')
+    const handleVideoReady = () => {
+        playerRef?.current?.seekTo(0,true);
+    }
+
+    useEffect(() => {
+        const loadLiveStreams = async () => {
+            try {
+                const liveStreamsResult = await runGraphQLQuery({ query: listLivestreams, variables: { filter: { date: { eq: today } } } })
+                liveStreamsResult.listLivestreams.items.map((event: LiveEvent) => {
+                    const rightNow = moment().utcOffset(moment().isDST() ? '-0400' : '-0500').format('HH:mm')
+                    const showTime = event?.startTime && event?.endTime && rightNow >= event.startTime && rightNow <= event.endTime
+                    if (showTime) {
+                        setcurrentEvent(event)
+                    }
+                })
+            }
+            catch (error) {
+                console.log(error)
+            }
+        }
+        loadLiveStreams();
+    }, [])
+
+    useEffect(() => {
+        const interval = setInterval(() => {
+            const start = currentEvent?.videoStartTime
+            const end = currentEvent?.endTime
+            const rightNow = moment().utcOffset(moment().isDST() ? '-0400' : '-0500').format('HH:mm')
+            //console.log(videoStartTime is ${currentEvent?.videoStartTime} endTime is ${currentEvent?.endTime} and current time is ${rightNow}`)
+            if (start && end) {
+                const showTime = rightNow >= start && rightNow <= end
+                if (showTime) {
+                    //console.log("ShowLive")
+                    setshowTime(true)
+                }
+            }
+            else {
+                setshowTime(false)
+            }
+        }, 1000);
+        return () => clearInterval(interval);
+    }, [currentEvent]);
+    // this page needs to be unmounted when navigating to teaching
+    return (
+        <Container style={{ backgroundColor: "black" }}>
+            <View style={style.player}>
+                {showTime ?
+                    <>
```

Comment:
Is there a reason why the `<YoutubePlayer/>` components are wrapped in Fragment?

Reply:
I originally had a title in there and forgot to remove it.